src/CVNBot/Project.cs

```diff
@@ -183,50 +171,97 @@ void GetNamespaces(bool snamespacesAlreadySet)
             }
         }
 
+        public struct MessagesOption
+        {
+            public int NumberOfArgs;
+            public string RegexName;
+            public bool NonStrictFlag;
+            public MessagesOption (int ArgNumberOfArgs, string ArgRegexName, bool ArgNonStrictFlag)
```

Comment:
I'm not 100% sure, but I think in CVNBot all properties and parameters start lowercase. Only methods and classes start uppercase.